config.py

```diff
@@ -60,7 +60,7 @@ class ProductionConfig(Config):
     @classmethod
     def init_app(cls, app):
         Config.init_app(app)
-
+        assert os.environ.get('SECRET_KEY'), 'SECRET_KEY IS NOT SET!'
```

Comment:
Ok good point. Can we replace that secure-looking string with something less secure (e.g. `'CHANGE ME'`)